src/lib/glocal/detect-exec-layout.ts

```diff
@@ -106,25 +87,22 @@ export function detectExecLayout(input: Input): ExecScenario {
       toolProject: false,
       toolCurrentlyPresentInNodeModules: false,
       runningLocalTool: false,
-      thisProcessToolBin,
+      thisProcessScriptPath,
       project: null,
     }
   }
 
   const projectNodeModulesDir = Path.join(projectDir, 'node_modules')
   const projectBinDir = Path.join(projectNodeModulesDir, '.bin')
-  const projectToolBinPath = Path.join(projectBinDir, thisProcessToolBin.name)
-  const project = {
+  const projectToolBinPath = Path.join(projectBinDir, input.depName)
```

Comment:
Problem here, an npm package dep name is not 1-to-1 with the bin name.

Reply:
Well, while glocal is theoretically designed for any library, it is only used for nexus. I'm happy to add an additional optional option that enables specifying the bin name, but we won't ever use it AFAIK

Reply:
Yeah ok, let's keep it iterating :)